pkg/services/correlations/database.go

```diff
@@ -55,6 +55,28 @@ func (s CorrelationsService) createCorrelation(ctx context.Context, cmd CreateCo
 	return correlation, nil
 }
 
+func (s CorrelationsService) deleteCorrelation(ctx context.Context, cmd DeleteCorrelationCommand) error {
+	return s.SQLStore.WithTransactionalDbSession(ctx, func(session *sqlstore.DBSession) error {
```

Comment:
I think here you don't really need WithTransactionalDbSession, WithDbSession is enough, since you do a get datasource which is a read only operation, then a delete. There is no fallback needed. Or I might miss something here.

Reply:
updated in 1621b197eb